src/components/Day.module.css

```diff
@@ -0,0 +1,7 @@
+.container{
+    display: grid;
+    justify-content: center;
+    align-content: center;
+    gap: 4px;
```

Comment:
With this gap you'll notice that now the user needs to click precisely at each bar to activate the drag. If you click between 2 bars, it won't work. We can bypass that using another strategy different than `gap`. I sent you a video to Discord explaining how.